packages/lsp7-contracts/package.json

```
@@ -46,9 +46,11 @@
     "lint:solidity": "solhint 'contracts/**/*.sol' && prettier --check 'contracts/**/*.sol'"
   },
   "dependencies": {
-    "@erc725/smart-contracts": "^7.0.0",
-    "@openzeppelin/contracts": "^4.9.3",
-    "@lukso/lsp1-contracts": "*",
+    "@erc725/smart-contracts-v8": "file:erc725-smart-contracts-v8-rc0.tgz",
```

Using local package here as well
```suggestion
    "@erc725/smart-contracts": "^7.0.0",
```